app/components/menu_item/index.tsx

```diff
@@ -83,25 +62,31 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     };
 });
 
+export type MenuItemProps = {
```

Comment:
how is this component different from `OptionsItem`?

Reply:
It's not different.  I refactored the code to use OptionsItem.  I will create another ticket to replace MenuItem occurences with OptionItem and then delete it from our codebase.